packages/libpsl.rb

```diff
@@ -3,19 +3,19 @@
 class Libpsl < Meson
   description 'C library for the Public Suffix List'
   homepage 'https://github.com/rockdaboot/libpsl'
-  @_ver = '0.21.2'
-  version @_ver.to_s
+  @_ver = '0.21.5'
+  version "#{@_ver}-1"
```

Comment:
How about just simply the following?
```
version '0.21.5'
...
git_hashtag version
```

Reply:
This is replacing existing "0.21.5" binaries...

Reply:
Ok that could work...

Reply:
Done.